src/main/scala/com/fulcrumgenomics/bam/EstimatePoolingFractions.scala

```diff
@@ -76,18 +79,24 @@ class EstimatePoolingFractions
   case class Locus(chrom: String, pos: Int, ref: Char, alt: Char, expectedSampleFractions: Array[Double], var observedFraction: Option[Double] = None)
 
   override def execute(): Unit = {
-    val vcfReader   = new VCFFileReader(vcf.toFile)
+    val vcfReader   = VcfSource(vcf)
```

Comment:
does this ever get closed?  Perhaps by the iterator below?

Reply:
Added a close().